ush/config.sh.RRFS_NA_13km

```diff
@@ -0,0 +1,87 @@
+MACHINE="jet"
+ACCOUNT="nrtrr"
+RESERVATION="rrfsdet"
+EXPT_BASEDIR="/home/rtrr/RRFS"
+EXPT_SUBDIR="RRFS_NA_13km"
+
+if [[ -n $RESERVATION ]] ; then
+  ACCOUNT=rtrr
+  PARTITION_DEFAULT=kjet
+  PARTITION_FCST=kjet
+  PARTITION_GRAPHICS=kjet
+fi
+
+FIXLAM_NCO_BASEDIR=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_lam.20210128
+FIXgsm=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_am.20210128
+GWD_HRRRsuite_BASEDIR=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_lam.20210128
+
+VERBOSE="TRUE"
+
+RUN_ENVIR="nco"
+PREEXISTING_DIR_METHOD="rename"
+
+PREDEF_GRID_NAME=GSD_RAP13km
+
+QUILTING="TRUE"
+CCPP_PHYS_SUITE="FV3_GSD_v0"
+FCST_LEN_HRS="12"
+LBC_SPEC_INTVL_HRS="6"
+
+HALO_BLEND=10
+
+WTIME_RUN_FCST="02:00:00"
+
+DATE_FIRST_CYCL="20210328"
+DATE_LAST_CYCL="20220328"
+CYCL_HRS=( "00" "06" "12" "18")
+
+EXTRN_MDL_NAME_ICS="FV3GFS"
+EXTRN_MDL_NAME_LBCS="FV3GFS"
+EXTRN_MDL_LBCS_FILE_NUM="14"  # 14th file is a 39-h forecast from GFS
+
+FV3GFS_FILE_FMT_ICS="grib2"
+FV3GFS_FILE_FMT_LBCS="grib2"
+
+envir="para"
+
+NET="RRFS_NA_13km"
+TAG="RRFS_NA_13km"
+
+USE_CUSTOM_POST_CONFIG_FILE="TRUE"
+CUSTOM_POST_CONFIG_FP="/mnt/lfs4/BMC/nrtrr/RRFS/dev1_13km_NA-ufs-srweather-app/src/EMC_post/parm/postxconfig-NT-fv3lam_rrfs.txt"
+ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_na_13km"
+NCARG_ROOT="/apps/ncl/6.5.0-CentOS6.10_64bit_nodap_gnu447"
+NCL_HOME="/home/rtrr/RRFS/graphics"
+NCL_REGION="conus"
```

Comment:
Should this be "north america"? I'm not sure since it's "conus" and "alaska" for the other two runs.

Reply:
Looks like our two NCL domain options are currently "conus" and "alaska".  May leave this as "conus" for now, but return to the issue of graphics production shortly.